script/setup

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
```

Comment:
`set -e` is your friend here (and in all bash files from now until eternity)

Reply:
oops!